### PR TITLE
[pylint] Bing (Videos) engine

### PR DESCRIPTION
**Pull request by return42**
## What does this PR do?

- Add Bing (Videos) engine to pylint process.
- Fix remarks from pylint and remove obsolete try/except block

## Why is this change important?

Code Review / no functional change!

## How to test this PR locally?

    make test.pylint

### CODE REVIEW

File: searx/engines/bing_videos.py [hunk 1]
```diff
@@ -28,74 +32,87 @@
...
-    params['cookies']['SRCHHPGUSR'] = \
+    params['cookies']['SRCHHPGUSR'] = (
         'ADLT=' + safesearch_types.get(params['safesearch'], 'DEMOTE')
+    )
 
     # language cookie
-    language = match_language(params['language'], supported_languages, language_aliases).lower()
+    language = match_language(
+        params['language'], supported_languages, language_aliases
+    ).lower()
```
Comment (dalf) [inline searx/engines/bing_videos.py]: I don't understand why the function is split into 3 lines.

Comment (return42) [reply searx/engines/bing_videos.py]: Just my subjective feeling of _readability_ / don't let us look to close, instead lets enforce https://github.com/searxng/searxng/pull/619 where we can define our Code-Style commitment.

**Comment by return42** [comment]
@dalf I rebased this PR (to get code style from black / https://github.com/searxng/searxng/pull/614#discussion_r772193521).

File: searx/engines/bing_videos.py [hunk 2]
```diff
@@ -82,22 +84,18 @@ def response(resp):
     dom = html.fromstring(resp.text)
 
     for result in dom.xpath('//div[@class="dg_u"]'):
-        try:
```
Comment (dalf) [inline searx/engines/bing_videos.py:85]: it is more than pylint, but why not.

Note that should be an issue: create a context manage named "no_exception":
```
for result in dom.xpath('//div[@class="dg_u"]'):
    with no_exception():
        ...
```

-->

exception are hidden EXCEPT in debug mode

**Review by dalf — APPROVED** [review]

File: searx/engines/bing_videos.py (hunk 2 above)
Comment (return42) [reply searx/engines/bing_videos.py:85]: > it is more than pylint, but why not.

more or less, it is pylint ...  [W0702 (bare-except)](http://pylint-messages.wikidot.com/messages:w0702)

> exception are hidden EXCEPT in debug mode

The problem of this try block is to catch all exception classes, what is an antipattern ...

>  create a context manage named "no_exception":

no please not .. it is a really bad practice to catch all issues / issues in the engines are handled by the caller of the engine.

I tested the bing engines and I can't find an exception, so for what is a try/except block needed? ..

**Pull request merged**